internal/dumper/dumper.go

```diff
@@ -424,10 +408,13 @@ func (d *Dumper) generatedFields(conn *Connection, table string) (map[string]boo
 		name := t[0].String()
 		extra := t[5].String()
 
-		// Can be either "VIRTUAL GENERATED" or "VIRTUAL STORED"
+		// Can be either "VIRTUAL GENERATED" or "STORED GENERATED"
 		// https://dev.mysql.com/doc/refman/8.0/en/show-columns.html
-		if strings.Contains(extra, "VIRTUAL") {
-			fields[name] = true
+		if strings.Contains(extra, "GENERATED") {
```

Comment:
I believe this has been a bug for a while, the valid values are "VIRTUAL GENERATED" or "STORED GENERATED".

Reported here: https://github.com/planetscale/cli/issues/645